mystyle-platform/mystyle-wp-custom-product-designer
# Set Window Title and Meta tags for design profile pages

1. Set meta via wordpress hook
2. Set meta via Rank Math plugin hook instead when its present
3. Set meta via Yoast SEO plugin hook instead when its present
4. Added design profile page title with design id and design title.

### All Submissions:







### How to test the changes in this Pull Request:

1. Install Yoast/Rankmath plugin 
2. Open any design profile page. 
3. View page source and check page title and meta description.

### Other information:




### Changelog entry
1. Set meta via wordpress hook
2. Set meta via Rank Math plugin hook instead when its present
3. Set meta via Yoast SEO plugin hook instead when its present
4. Added design profile page title with design id and design title.
